src/app/Header.tsx

```diff
@@ -34,151 +25,57 @@ type Props = {
-            <Nav id="search-nav" activeKey={base} className={styles.navmain}>
-              <Nav.Item>
-                <Nav.Link
-                  id="works-link"
-                  eventKey={'/doi.org'}
-                  href={'/doi.org'}
-                  className="text-nowrap"
-                >
-                  <FontAwesomeIcon icon={faNewspaper} /> Works
-                </Nav.Link>
-              </Nav.Item>
-              <Nav.Item>
-                <Nav.Link
-                  id="people-link"
-                  eventKey={'/orcid.org'}
-                  href={'/orcid.org'}
-                  className="text-nowrap"
-                >
-                  <FontAwesomeIcon icon={faUserGraduate} /> People
-                </Nav.Link>
-              </Nav.Item>
-              <Nav.Item>
-                <Nav.Link
-                  id="organizations-link"
-                  eventKey={'/ror.org'}
-                  href={'/ror.org'}
-                  className="text-nowrap"
-                >
-                  <FontAwesomeIcon icon={faUniversity} /> Organizations
-                </Nav.Link>
-              </Nav.Item>
-              <Nav.Item>
-                <Nav.Link
-                  id="repositories-link"
-                  eventKey={'/repositories'}
-                  href={'/repositories'}
-                  className="text-nowrap"
-                >
-                  <FontAwesomeIcon icon={faDatabase} /> Repositories
-                </Nav.Link>
-              </Nav.Item>
-            </Nav>
+            <Search />
+            <NavLinks>
+              <WorksLink />
+              <PeopleLink />
+              <OrganizationsLink />
+              <RepositoriesLink />
+            </NavLinks>
           </Form>
+
+          {/* Pages dropdown, support button, and right content */}
           <Nav className={`hidden-mobile ${styles.navright} mt-3`}>
             <NavDropdown
-              // eventKey={1}
               title="Pages"
               id="pages-dropdown"
               data-cy="pages"
               className={styles.navdropdown}
             >
-              <NavDropdown.Item eventKey="1.1" data-cy="about" href="/about">
-                About
-              </NavDropdown.Item>
-              <NavDropdown.Item eventKey="1.2" data-cy="statistics" href="/statistics">
-                Statistics
-              </NavDropdown.Item>
+              <AboutLink />
+              <StatisticsLink />
             </NavDropdown>
-            <Nav.Item>
-              <Nav.Link
-                eventKey={2}
-                data-cy="support"
-                href="https://support.datacite.org/docs/datacite-commons"
-                target="_blank"
-                rel="noreferrer"
-              >
-                Support
-              </Nav.Link>
-            </Nav.Item>
-            {user && (
-              <NavDropdown title={user.name} id="basic-nav-dropdown">
-                {user.beta_tester && (
```

Comment:
@jrhoads Is this okay to remove?

Reply:
Yeah.  We already dropped the beta path in the migration to the app router.